src/features/DisplayControls/NursingTasks/utils/NursingTasksUtils.js

```diff
@@ -255,10 +255,7 @@ export const disableTaskTilePastNextSlotTime = (
     const upcomingTaskTimeInEpoch = upcomingTask.startTimeInEpochSeconds;
     const currentTimeInEpoch = moment().unix();
     if (upcomingTaskTimeInEpoch <= currentTimeInEpoch) {
-      if (
-        currentTask.isANonMedicationTask &&
-        currentTask.taskType !== "nursing_activity"
-      ) {
+      if (currentTask.taskType !== "nursing_activity") {
```

Comment:
Instead of the If else block, We can directly do currentTask.isDisabled = currentTask.taskType === "nursing_activity"